www/src/App.js

```diff
@@ -2,33 +2,14 @@ import 'semantic-ui-css/semantic.min.css';
 import React, { Component } from 'react';
 import { Link, Route, Switch } from 'react-router-dom';
 import { Menu } from 'semantic-ui-react'
-import Home from './Home';
+import HomeApp from './pages/home/HomeApp';
+import UserApp from './pages/user/UserApp';
```

Comment:
I'm calling them apps but it can me be a module or a service whatever name you want. However I want these to be fully isolated applications because I fully intend on reusing this project once I complete these common use cases for the marketplace.